x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts

```diff
@@ -80,25 +79,64 @@ describe.skip('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
       .should('have.text', createdHost.hostname);
   });
 
+  it('should update endpoint policy on Endpoint', () => {
+    const parseRevNumber = (revString: string) => Number(revString.match(/\d+/)?.[0]);
+
+    loadPage(APP_ENDPOINTS_PATH);
+
+    cy.getByTestSubj('policyListRevNo')
+      .first()
+      .invoke('text')
+      .then(parseRevNumber)
+      .then((initialRevisionNumber) => {
+        // Update policy
+        cy.getByTestSubj('policyNameCellLink').first().should('exist').click();
+
+        cy.getByTestSubj('endpointPolicyForm-malware-blocklist-enableDisableSwitch').as(
+          'blocklistSwitch'
+        );
+        cy.get('@blocklistSwitch').should('exist').click();
+        cy.get('@blocklistSwitch').should('be.enabled');
+
+        cy.getByTestSubj('policyDetailsSaveButton').should('be.visible').click();
+        cy.getByTestSubj('policyDetailsConfirmModal').should('be.visible').click();
+        cy.getByTestSubj('confirmModalConfirmButton').should('be.visible').click();
+        cy.contains(/has been updated/);
+
+        cy.getByTestSubj('policyDetailsBackLink').should('be.visible').click();
+
+        // Assert disappearing 'Out-of-date' indicator, Success Policy Status and increased revision number
+        cy.getByTestSubj('rowPolicyOutOfDate', { timeout: 10000 }).should('not.exist'); // depends on the 10s auto-refresh
+
+        cy.getByTestSubj('policyStatusCellLink').first().should('contain', 'Success');
+
+        cy.getByTestSubj('policyListRevNo')
+          .first()
+          .should(($div) => {
+            const revNumber = parseRevNumber($div.text());
+            expect(revNumber).to.equal(initialRevisionNumber + 1);
+          });
+      });
+  });
+
```

Comment:
I see that you moved this around with some modifications but if you are up to I would refactor this test a little. I believe we could try to split this up into several sections instead of chaining and base it off api calls with final validation of revision number being bumped in the ui. This could go like:
1. Navigate to `/app/security/administration/policy/xxx/settings`
2. Intercept `/api/fleet/package_policies/xxx` and save `revision` from the response
3. Do manual update to validate the ui aspect of bumping revision number
4. Navigate to `/app/security/administration/endpoints` and validate that revision number 

Let me know what you think

Reply:
Thanks for the thoughts. I do like the idea of breaking it down into smaller bits but if we were to start from `/app/security/administration/policy/xxx/settings` we would lose testing whether the UI navigation from endpoint list works. I don't think it's a giant loss but still a loss.